src/fracTrackerMobileLayer.js

```diff
@@ -49,13 +48,18 @@ L.GeoJSON.FracTrackerMobile = L.GeoJSON.extend(
         }
 
         return $.getJSON(fractrackerMobile_url);
-
       })().done(function(data) {
         self.parseData(data);
         if (typeof self._map.spin === 'function') {
           self._map.spin(false);
+          clearTimeout(timeout);
```

Comment:
`timeout` will be `undefined` here.  To avoid this the `setTimeout` function can be moved to L49. The `timeout` variable needs to be accessible here.

Reply:
Instead of using the timeout can we chain a .fail to the function, that will stop the spinner if the call fails.

Reply:
Yes. You are right. The suggestion you have made has been implemented in PR #417. You could take a look at it. I did come across instances of data taking a long time to load. Your implementation here would help in those cases. But let's put a hold on this for now.

Reply:
Okay.

Reply:
Alright, I've moved the timeout function to L49 as you requested.